Description: Enable set_orientation for WnckTasklist
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 gnome-python-desktop (2.32.0+dfsg-zevenos2) unstable; urgency=low
 .
   * Added set_orientation for WnckTasklist
Author: Leszek Lesner <leszek@zevenos.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- gnome-python-desktop-2.32.0+dfsg.orig/wnck/wnck.defs
+++ gnome-python-desktop-2.32.0+dfsg/wnck/wnck.defs
@@ -724,6 +724,15 @@
   (return-type "gint")
 )
 
+(define-method set_orientation
+  (of-object "WnckTasklist")
+  (c-name "wnck_tasklist_set_orientation")
+  (return-type "none")
+  (parameters
+    '("GtkOrientation" "orientation")
+  )
+)
+
 (define-method set_icon_loader
   (of-object "WnckTasklist")
   (c-name "wnck_tasklist_set_icon_loader")
